mudlet-mapper.xml

```diff
@@ -6739,6 +6739,15 @@ end
   return c
 end
 
+function mmp.changeEchoColour()
+  if color_table[mmp.settings.echocolour] then
+    mmp.echo(&quot;Echo colour is now &quot;..mmp.settings.echocolour)
+  else
+    mmp.echo(&quot;Not a valid cecho color - Setting to default&quot;)
```

Comment:
I'd say keeping the old one is best - would be annoying if you get the colour wrong and your old one is reset